code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -825,6 +825,33 @@
 	owner.visible_message("<span class='danger'>[owner] is protected by a ring of [ghost_counter] ghosts!</span>")
 	return ..()
 
+//Fire-Tail
+
+/obj/item/melee/fire_tail
+	name = "drake tail"
+	desc = "A long flaming tail from an ash drake."
+	icon_state = "draketail"
+	item_state = "chain"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	damtype = "fire"
+	force = 20
+	throwforce = 20
```

Comment:
Whips are not really good throwing weapons, Trilby, give it a less senseless value.